src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -1755,27 +1823,31 @@ def _process_decln(self, scope, symbol_table, decl, visibility_map=None):
         :type symbol_table: py:class:`psyclone.psyir.symbols.SymbolTable`
         :param decl: fparser2 parse tree of declaration to process.
         :type decl: :py:class:`fparser.two.Fortran2003.Type_Declaration_Stmt`
-        :param visibility_map: mapping of symbol name to visibility (for \
+        :param visibility_map: mapping of symbol name to visibility (for
             those symbols listed in an accessibility statement).
-        :type visibility_map: dict with str keys and \
+        :type visibility_map: dict with str keys and
             :py:class:`psyclone.psyir.symbols.Symbol.Visibility` values
+        :param statics_list: the names of symbols which are static (due to
+            appearing in a SAVE statement). If all symbols are static then
+            this contains the single entry "*".
+        :type statics_list: Optional[Iterable[str]]
```

Comment:
I don't think Optional goes here. Optional is a shorthand notation for Union[X, None], but we don't accept None anymore.

The fact that it is an optional argument for the caller because it has a default value is a unfortunate name clash, but this is not specified on the type hints/docstrings.

Reply:
Good point.